styles/properties.less

```diff
@@ -112,7 +118,7 @@
     flex-direction: row;
     align-items: center;
 
-    > .bpp-error-message {
+    >.bpp-error-message {
```

Comment:
What and why do we reformat the whole file?

Reply:
lol I committed only what I wanted to remove 🙈

Reply:
It should be OK now. Thanks for the review!